src/controllers/auth/AuthController.ts

```
@@ -0,0 +1,64 @@
+      throw new BadRequestError('PLATFORM_BASE_URL is not configured')
+    }
+
+    const response = await axios.post<OrgTokenResponse>(
+      `${platformBaseUrl}/v1/orgs/${orgId}/token`,
+      { clientId: body.clientId, clientSecret: body.clientSecret },
+      { headers: { 'Content-Type': 'application/json', accept: 'application/json' } },
+    )
+
+    return response.data
+  }
+// TODO: Remove these test endpoints after manual testing is done
```

ESLint

[error] Insert `··`

(prettier/prettier)